src/main/java/de/tum/in/www1/artemis/web/rest/LegacyResource.java

```diff
@@ -26,37 +19,15 @@
 @Deprecated(forRemoval = true)
 public class LegacyResource {
 
-    private final PublicLtiResource publicLtiResource;
-
     private final PublicProgrammingSubmissionResource publicProgrammingSubmissionResource;
 
     private final PublicResultResource publicResultResource;
 
-    public LegacyResource(PublicLtiResource publicLtiResource, PublicProgrammingSubmissionResource publicProgrammingSubmissionResource, PublicResultResource publicResultResource) {
-        this.publicLtiResource = publicLtiResource;
+    public LegacyResource(PublicProgrammingSubmissionResource publicProgrammingSubmissionResource, PublicResultResource publicResultResource) {
         this.publicProgrammingSubmissionResource = publicProgrammingSubmissionResource;
         this.publicResultResource = publicResultResource;
     }
 
-    /**
-     * POST lti/launch/:exerciseId : Launch the exercise app using request by an LTI consumer. Redirects the user to
-     * the exercise on success.
-     *
-     * @param launchRequest the LTI launch request (ExerciseLtiConfigurationDTO)
-     * @param exerciseId    the id of the exercise the user wants to open
-     * @param request       the request
-     * @param response      the response
-     * @deprecated use {@link PublicLtiResource#launch(LtiLaunchRequestDTO, Long, HttpServletRequest, HttpServletResponse)} instead
-     */
-    @PostMapping("lti/launch/{exerciseId}")
```

Comment:
Are you aware that we mentioned in the release that this won't be removed before June 2024?